example/tools/format/BUILD.bazel

```diff
@@ -14,6 +14,7 @@ prettier.prettier_binary(
     name = "prettier",
     # Allow the binary to be run outside bazel
     env = {"BAZEL_BINDIR": "."},
+    fixed_args = ["--log-level=silent"],
```

Comment:
nice! you may want to set it to error for logging errors on failure. Haven't tested the difference, but silent might be too silent.